samples/templates/express-direct-auth/web-server/views/login-with-widget.mustache

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- okta-signin-widget assets are avilable on CDN -->
+    <script src="https://global.oktacdn.com/okta-signin-widget/5.5.2/js/okta-sign-in.min.js" type="text/javascript"></script>
```

Comment:
Use `{{ siwVersion }}`, all samples should run against the same widget version (samples/config.js)